texts/formatted_harry_potter.txt

```diff
@@ -0,0 +1,1679 @@
+;	"struggle" vblex pres
+;	"struggle" vblex imp
+"<against>"
+	"against" adj
+	"against" adv
+	"against" pr
+"<Lord>"
+	"lord" n sg
+;	"lord" vblex inf
+;	"lord" vblex pres
+;	"lord" vblex imp
+;	"Lord" np cog sg
+"<Voldemort>"
+	"Voldemort" np cog sg
+;	"Voldemort" np ant m sg
+"<,>"
+	"," cm
+"<a>"
+	"a" det ind sg
+"<dark>"
+;	"dark" n sg
+	"dark" adj sint
+"<wizard>"
+	"wizard" n sg
+"<who>"
+;	"who" prn itg mf sp
+	"who" prn rel an mf sp
+"<intends>"
+	"intend" vblex pres p3 sg
+"<to>"
+	"to" pr
+"<become>"
+	"become" vblex inf
+;	"become" vblex pres
+;	"become" vblex pp
+;	"become" vblex imp
+"<immortal>"
+	"immortal" adj
+"<,>"
+	"," cm
+"<overthrow>"
+;	"overthrow" n sg
+	"overthrow" vblex inf
+;	"overthrow" vblex pres
+;	"overthrow" vblex imp
+"<the>"
+	"the" det def sp
+"<wizard>"
+	"wizard" n sg
+"<governing>"
+	"governing" adj
+;	"govern" vblex pprs
+;	"govern" vblex subs
+;	"govern" vblex ger
+"<body>"
+	"body" n sg
+"<known>"
+;	"known" adj
+	"know" vblex pp
+"<as>"
+;	"as" adv
+;	"as" preadv
+;	"as" cnjsub
+;	"as" cnjadv
+	"as" pr
+"<the>"
+	"the" det def sp
+"<Ministry>"
+	"ministry" n sg
+"<of>"
+	"of" pr
+"<Magic>"
+;	"magic" adj
+	"magic" n sg
+"<and>"
+	"and" cnjcoo
+"<subjugate>"
+	"subjugate" vblex inf
+;	"subjugate" vblex pres
+;	"subjugate" vblex imp
+"<all>"
+;	"all" adv
+;	"all" predet sp
+	"all" det qnt pl
+;	"all" prn qnt mf sp
+"<wizards>"
+	"wizard" n pl
+"<and>"
+	"and" cnjcoo
+"<Muggles>"
+	"*Muggles"
+"<(>"
+	"(" lpar
+"<non>"
+	"non" adv
+"<->"
+	"-" guio
+"<magical>"
+	"magical" adj
+"<people>"
+;	"people" n sg
+	"person" n pl
+"<)>"
+	")" rpar
+"<.>"
+	"." sent
+"<Since>"
+;	"since" adv
+;	"since" cnjsub
+	"since" pr
+"<the>"
+	"the" det def sp
+"<release>"
+;	"release" n sg
+	"release" vblex inf
+;	"release" vblex pres
+;	"release" vblex imp
+"<of>"
+	"of" pr
+"<the>"
+	"the" det def sp
+"<first>"
+;	"first" adv
+	"first" adj ord
+"<novel>"
+;	"novel" adj
+	"novel" n sg
+"<,>"
+	"," cm
+"<Harry>"
+	"Harry" np ant m sg
+"<Potter>"
+;	"potter" n sg
+	"Potter" np cog sg
+"<and>"
+	"and" cnjcoo
+"<the>"
+	"the" det def sp
+"<Philosopher>"
+	"philosopher" n sg
+"<'s>"
+	"'s" gen
+;	"be" vbser pres p3 sg
+;	"have" vbhaver pres p3 sg
+"<Stone>"
+	"stone" n sg
+;	"stone" vblex inf
+;	"stone" vblex pres
+;	"stone" vblex imp
+;	"Stone" np cog sg
+;	"Stone" np ant m sg
+"<,>"
+	"," cm
+"<on>"
+;	"on" adv
+	"on" pr
+"<26>"
+	"26" num
+"<June>"
+	"June" n sg
+"<1997>"
+	"1997" num
+"<,>"
+	"," cm
+"<the>"
+	"the" det def sp
+"<books>"
+	"book" n pl
+;	"book" vblex pres p3 sg
+"<have>"
+;	"have" vbhaver inf
+	"have" vbhaver pres
+;	"have" vblex inf
+;	"have" vblex pres
+;	"have" vblex imp
+"<found>"
+;	"found" adj
+;	"find" vblex past
+	"find" vblex pp
+;	"found" vblex inf
+;	"found" vblex pres
+;	"found" vblex imp
+"<immense>"
+	"immense" adj
+"<popularity>"
+	"popularity" n sg
+"<,>"
+	"," cm
+"<critical>"
+	"critical" adj
+"<acclaim>"
+	"acclaim" vblex inf
+;	"acclaim" vblex pres
+;	"acclaim" vblex imp
```

Comment:
should be a noun, I think, though that isn't listed

Reply:
Yeah i leaved as it not found

Reply:
What to do should i have to add that?

Reply:
I would add it to the list here